packages/components/src/base_tooltip/index.jsx

```diff
@@ -53,4 +53,5 @@ BaseTooltip.propTypes = {
   arrowClassName: PropTypes.string,
 };
 
+export * from './constants';
```

Comment:
Please check how constants are named. We export named constants 1:1, so we need easily identifiable and collision free names

Reply:
I think it could make sense to prefix all of them with 'TOOLTIP_'